recipes/perl-module-implementation/build.sh

```diff
@@ -3,10 +3,10 @@
 # If it has Build.PL use that, otherwise use Makefile.PL
 if [ -f Build.PL ]; then
     perl Build.PL
-    ./Build
-    ./Build test
+    perl ./Build
```

Comment:
I have done this now already multiple time, if you think we should do this for all packages I would be ok to merge a PR for `sed`ding all perl pkgs :)